bin/ci-stack.ts

```diff
@@ -7,12 +7,28 @@
  */
 
 import 'source-map-support/register';
-import { App } from '@aws-cdk/core';
+import { App, RemovalPolicy } from '@aws-cdk/core';
 import { CIStack } from '../lib/ci-stack';
 import { CIConfigStack } from '../lib/ci-config-stack';
+import { DeployAWSAssets } from '../lib/DeployAWSAssets';
 
 const app = new App();
 
-new CIConfigStack(app, 'CI-Config-Dev', {});
+const env: string = 'Dev';
 
-new CIStack(app, 'CI-Dev', { });
+const ciConfigStack = new CIConfigStack(app, `OpenSearch-CI-Config-${env}`, {});
+
+const ciStack = new CIStack(app, `OpenSearch-CI-${env}`, {
+  ecrAccountId: ciConfigStack.account,
+});
+
+new DeployAWSAssets(app, `OpenSearch-CI-Deploy-Assets-${env}`, {
+  removalPolicy: RemovalPolicy.DESTROY,
```

Comment:
We do not want the Assests to be destroyed not matter what. Please remove this policy assuming default is retain

Reply:
The default policy is Retain. I mentioned destroy here since this deployment is only used for dev envs. https://github.com/opensearch-project/opensearch-ci/pull/77/files#diff-d638b1fcda588e6ca1f48230caf1d17595dccafad1e580bcd937a15150343471R24

Reply:
Not necessarily. We say that this code is production if any external users want to deploy the stack. If they are using this code base it will easily be removed. Better retain like we have for other assets like s3 auditing bucket, etc.

Reply:
This code is production, but by no means is these are the properties used in production. If we retain the assets, the redeployment fails since cloud formation does not update the ecr repository resource and would require user to delete everything manually each time.

Reply:
Thats fine, the user can manually delete it but here is a use case that if we delete the stack by mistake or something happens, we want to be retain any ECR repo since those are public. That should never be taken down since it's one of the release channel now.